packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -85,6 +86,11 @@ const propTypes = {
    * Whether or not the legend is visible. Use this props to hide a legend while still creating it on the DOM for accessibility.
    */
   isLegendHidden: PropTypes.bool,
+
+  /**
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the @private syntax so this doesn't show up in the generated prop-types documentation.
```suggestion
   * @private
   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```